### DISCUSSION TITLE
Facing Error

### DISCUSSION THREAD

**Discussion by SanjayKumar-M** [jaykali/maskphish#107] [Q&A] [answered]
When I clone this repo in my ubuntu system and executing the command I'm facing the error like this
maskphish.sh: line 27: curl: command not found
How can I over come this ?? 

![Screenshot from 2022-11-13 23-20-26](https://user-images.githubusercontent.com/73515250/201536560-eb09a3bc-6adc-437b-9eee-05daaa3ad5cc.png)

**Comment by jaykali** [comment, marked as answer]
`sudo apt install curl`
Then run MaskPhish again

**Reply by jaykali** [reply]
> `sudo apt install curl`
> Then run MaskPhish again

Hope this solve your issue.

**Reply by SanjayKumar-M** [reply]
Yeah solved thanks :)
